projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -67,18 +67,20 @@ export class AuthService implements OnDestroy {
    * Emits details about the authenticated user when `isAuthenticated$` is `true`.
    */
   readonly user$ = this.isAuthenticated$.pipe(
-    filter((authenticated) => authenticated),
     distinctUntilChanged(),
```

Comment:
Do you still need this line if you're no longer filtering on `authenticated`? Just curious.

Reply:
Previously, this line had no use. We filtered to only get `true`, then we add `distinctUntilChanged`, but it would never change coz of the filter.

Now it does have use as we are allowing every emit to go through.